bind/gencffi_cdef.go

```diff
@@ -5,18 +5,22 @@
 package bind
 
 import (
+	"fmt"
 	"go/types"
 	"strconv"
 	"strings"
 )
 
+// Generate C definitions of Go struct.
```

Comment:
the canonical way to document functions/methods is:

```go
// genCdefStruct generates C definitions for a Go struct.
func (g *cffiGen) genCdefStruct(s Struct) {
```

(ie: `// [method or function name] [verb] [stuff...]`)